<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
    <widget class="QWidget" name="centralwidget">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
        <item>
         <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>数独游戏</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="styleSheet">
             <string>font-size:24px; font-weight:bold;</string>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="continueButton">
            <property name="text">
             <string>继续游戏</string>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="newGameButton">
            <property name="text">
             <string>新的开始</string>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearRecordsButton">
            <property name="text">
             <string>清空记录</string>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitButton">
            <property name="text">
             <string>退出游戏</string>
            </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
                <width>20</width>
                <height>40</height>
             </size>
            </property>
         </spacer>
        </item>
     </layout>
    </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
